dynamic-storage.html.md.erb

```diff
@@ -179,3 +179,123 @@ To get more information about the underlying EMC ECS product, please consult the
 * GET Object torrent
 
 Using a S3 compatible URL, Dynamic Storage can easily be used as a backend.
+
+#### Best Practices
+
+#### App Design
+
+•	Don’t require low latency
```

Comment:
Use asterisks with one space for bullet points (`* this is a bullet point` instead of `•        this is a bullet point`).